resources/guest_configs/microvm-kernel-ci-aarch64-4.14.config

```diff
@@ -2085,7 +2085,7 @@ CONFIG_PRINTK_TIME=y
 CONFIG_CONSOLE_LOGLEVEL_DEFAULT=7
 CONFIG_MESSAGE_LOGLEVEL_DEFAULT=4
 # CONFIG_BOOT_PRINTK_DELAY is not set
-CONFIG_DYNAMIC_DEBUG=y
+CONFIG_DYNAMIC_DEBUG=n
```

Comment:
This might be useful if we want to enable `pr_debug` at runtime.

Reply:
Oh that is very cool! https://www.kernel.org/doc/html/v4.11/admin-guide/dynamic-debug-howto.html

I may have been overzealous removing options. The trimming work was from when I was regenerating the CI artifacts and I was trying to get faster boottimes. Eventually it wasn't needed.